docs/team/wangwaynesg.md

```diff
@@ -1,7 +1,56 @@
 
-# Wayne Wang - Project Portfolio Page
+# Wang Wayne - Project Portfolio Page
 
 ## Overview
 
+eCardnomics is a **desktop flashcard application to quickly create, manage, and access new flashcards via a Command
+Line Interface (CLI)**. While it is mainly targeted at Junior College economics students, eCardnomics can be used
+by anyone who would need to keep track of large amounts of *text-based* information. This is facilitated by the 
+capabilities to group similar flashcards into decks and tag decks to provide a summary of the information contained
+in the deck. Furthermore, the game mode and the feature of exporting decks into powerpoint allow review of the 
+help theinformation to user to memorize the text data.
```

Comment:
Should the overview be rephrased so that it does not resemble the one that I wrote?

Take note [Draft the PPP -> Sections to include -> Overview](https://nus-cs2113-ay2021s1.github.io/website/schedule/week12/project.html)

Reply:
Thanks for notifying! Will make changes accordingly!